docs/source/getting_started/active_learning.rst

```diff
@@ -41,21 +41,21 @@ Lightly makes use of the following concepts for active learning:
    to sample from. Furthermore, one can query it to get a new batch of images.
    To initialize an `ActiveLearningAgent` you need an `ApiWorkflowClient`.
    
-* **SamplerConfig:** :py:class:`lightly.active_learning.config.sampler_config.SamplerConfig`
-   The `SamplerConfig` allows the configuration of a sampling request. In particular,
+* **SelectionConfig:** :py:class:`lightly.active_learning.config.selection_config.SelectionConfig`
+   The `SelectionConfig` allows the configuration of a selection request. In particular,
    you can set number of samples, the name of the resulting selection, and the `SamplingMethod`.
    Currently, you can set the `SamplingMethod` to one of the following:
 
-   * Random: Selects samples uniformly at random.
-   * Coreset: Greedily selects samples which are diverse.
-   * Coral: Combines Coreset with scores to do active learning.
+   * RANDOM: Selects samples uniformly at random.
+   * CORESET: Greedily selects samples which are diverse.
+   * CORAL: Combines CORESET with scores to do active learning.
```

Comment:
small or big?! further up its always small.. now its big again?

Reply:
Always BIG in the normal docs, Small in the docker docs.